### PR TITLE
 making clearing button  clear the newValue variable also

### PR DESCRIPTION
**Pull request by yaaHosein**

### CODE REVIEW

File: calculator.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ document.getElementById('button0').innerHTML = "0";
 
 clearingButton.addEventListener("click", function() {
     result.textContent = '';
+    lastValue = ""
```
Comment (SMH110) [inline calculator.js]: It makes sense to reset the value of the operator variable also 

```js 
operator = null
 ```

Comment (yaaHosein) [reply calculator.js]: I did, but why!?

Comment (SMH110) [reply calculator.js]: Because the operator value was falsy when the app started.

**Review by SMH110 — APPROVED** [review]

File: calculator.js (hunk 1 above)
Comment (yaaHosein) [reply calculator.js]: Ok thanks !!

**Pull request merged**